corgi/monitor/consumer.py

```diff
@@ -77,19 +84,69 @@ def on_message(self, event: Event):
             self.accept(event.delivery)
 
 
+class BrewTagUMBReceiverHandler(UMBReceiverHandler):
+    """Handle messages about Brew builds that have tags added or removed"""
+
+    def on_message(self, event: Event) -> None:
+        logger.info("Received UMB event on %s: %s", self.virtual_topic_address, event.message.id)
+        message = json.loads(event.message.body)
+        build_id = message["build"]["build_id"]
+
+        tag_added_or_removed = message["tag"]["name"]
+        if self.virtual_topic_address.endswith(".tag"):
```

Comment:
Can this instead check the class name (or type) instead of an arbitrary part of the virtual topic?

Reply:
Gotcha, keep it as is then.